### PR TITLE
25 update dependencies to latest version

### PR DESCRIPTION
**Pull request by Swissbite**
- Updated dependencies to the `@latest` of each version (Date of today)
- Fix lint error on `src/contentful/import-management.js`
- Enforce `lint` on gitlab actions definition
- Enforce engines with `.npmrc` config

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -17,10 +17,11 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: npm install, build, and test
+    - name: npm install, build, test and lint
       run: |
         npm install
         npm run build --if-present
         npm test
+        npm run lint
```
Comment (lotas) [inline .github/workflows/nodejs.yml:25]: :+1: Wouldn't it be better to lint before running the test?

**Comment by lotas** [comment]
Thanks a lot! :+1:

**Comment by Swissbite** [comment]
Does it matter? Both must run.Am 22.11.2020 21:53 schrieb Yaraslau Kurmyza <notifications@github.com>:

@lotas commented on this pull request.







In .github/workflows/nodejs.yml:

>        run: |

         npm install

         npm run build --if-present

         npm test

+        npm run lint



👍 Wouldn't it be better to lint before running the test?

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (Swissbite) [reply .github/workflows/nodejs.yml:25]: Does it matter? Both must run.
But if you wish that I change the order, then I'll do it :-)

Comment (lotas) [reply .github/workflows/nodejs.yml:25]: > Does it matter? Both must run.

I've always thought that this is the right order. First you run "cheaper" lint and fail early if there are errors, and only then run more expensive tests. 

But it's fine, the codebase is not that big :)  
Was just wondering

Comment (Swissbite) [reply .github/workflows/nodejs.yml:25]: This is one way of defining the order.
I thought that a proper working test is more important than a proper lint. Therefore, I test the critical stuff first.

Comment (lotas) [reply .github/workflows/nodejs.yml:25]: Agree. As long as both run :+1:

**Pull request merged**